cfpb/consumerfinance.gov
# Update Rental Assistance Tool to version 0.3.2

This PR updates our Rental Assistance Finder Tool to version 0.3.2, which fixes an accessibility error.

See [this release](https://github.com/cfpb/rental-assistance-finder/releases/tag/0.3.2) for more information.

## Changes
- Update `rental-assistance-finder` to version `0.3.2`

## How to test this PR
1. Pull down the PR
2. Visit [the RAF Tool](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/)
3. Use the WAVE tool to check for contrast accessibility errors
## Checklist

#### Browser testing

#### Accessibility

#### Other
